Automattic/wp-calypso
# Checklist: Add a launch site task to the checklist

#### Changes proposed in this Pull Request

* This adds a "Launch site" task to the checklist. It needs to be tested in conjunction with D20479-code.

#### Testing instructions

* Apply D20479-code to your sandbox
* Open an incognito window
* Create a new account and site from wordpress.com/start/private
* Open the checklist for the new site: calypso.localhost:3000/checklist/:site
* Check that you don't see a "Launch site" task as your email hasn't been verified
* Verify your email address
* Check that you now see a "Launch site" task at the bottom of your checklist
* Click the "Launch site" button
* Check that the "Launch site" task is now marked as complete
* Check that your site is now launched
